src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -52,9 +52,9 @@ def _missing_(cls, version):
                     return FluentVersion(version)
             else:
                 raise RuntimeError(
-                    f"The passed version '{version[:-2]}' does not exist."
-                    f" Available version strings are: "
-                    f"{[ver.value for ver in FluentVersion]} "
+                    f"The specified version '{version[:-2]}' is not supported."
+                    + f" Supported versions include: "
```

Comment:
"include" suggests that what follows might be a subset of all supported versions.

Reply:
You are right, that was my intention, considering that pyfluent does work for example with Fluent v241 currently.

However, as the code blocks execution if the version is not on that list, I am going to change the word 'include' to 'are' as I agree it is more appropriate currently.